exporting/exporting.conf

```diff
@@ -0,0 +1,88 @@
+[exporting:global]
+    enabled = no
+    # send configured labels = yes
+    # send automatic labels = no
+    # update every = 10
+
+[prometheus:exporter]
+    # send names instead of ids = yes
+    # send configured labels = yes
+    # send automatic labels = no
+    # send charts matching = *
+    # send hosts matching = localhost *
+    # prefix = netdata
+
+# An example configuration for graphite, json, opentsdb exporting connectors
+# [graphite:my_graphite_instance]
+    # enabled = no
+    # destination = localhost
+    # data source = average
+    # prefix = netdata
+    # hostname = my_hostname
+    # update every = 10
+    # buffer on failures = 10
+    # timeout ms = 20000
+    # send names instead of ids = yes
+    # send charts matching = *
+    # send hosts matching = localhost *
+
+# [prometheus_remote_write:my_prometheus_remote_write_instance]
+    # enabled = no
+    # destination = localhost
+    # remote write URL path = /receive
+    # data source = average
+    # prefix = netdata
+    # hostname = my_hostname
+    # update every = 10
+    # buffer on failures = 10
+    # timeout ms = 20000
+    # send names instead of ids = yes
+    # send charts matching = *
+    # send hosts matching = localhost *
+
+# [kinesis:my_kinesis_instance]
+    # enabled = no
+    # destination = us-east-1
+    # stream name = netdata
+    # aws_access_key_id = my_access_key_id
+    # aws_secret_access_key = my_aws_secret_access_key
+    # data source = average
+    # prefix = netdata
+    # hostname = my_hostname
+    # update every = 10
+    # buffer on failures = 10
+    # timeout ms = 20000
+    # send names instead of ids = yes
+    # send charts matching = *
+    # send hosts matching = localhost *
+
+# [pubsub:my_pubsub_instance]
+    # enabled = no
+    # destination = pubsub.googleapis.com
+    # credentials file = /etc/netdata/pubsub_credentials.json
+    # project id = my_project
+    # topic id = my_topic
+    # data source = average
+    # prefix = netdata
+    # hostname = my_hostname
+    # update every = 10
+    # buffer on failures = 10
+    # timeout ms = 20000
+    # send names instead of ids = yes
+    # send charts matching = *
+    # send hosts matching = localhost *
+
+# [mongodb:my_mongodb_instance]
+    # enabled = no
+    # destination = localhost
+    # database = my_database
+    # collection = my_collection
+    # data source = average
+    # prefix = netdata
+    # hostname = my_hostname
+    # update every = 10
+    # buffer on failures = 10
+    # timeout ms = 20000
+    # send names instead of ids = yes
+    # send charts matching = *
+    # send hosts matching = localhost *
```

Comment:
We are missing examples for `JSON` and `OpenTSDB`.

Reply:
They are the same as graphite and I added
> \# An example configuration for graphite, json, opentsdb exporting connectors

Thus, we already have an example.